_tuning-your-cluster/availability-and-recovery/segment-replication/index.md

```diff
@@ -28,6 +28,13 @@ Segment replication can be applied in a variety of scenarios, including:
 - When experiencing very high loads, you want to add new nodes but don't want to index all data immediately.
 - OpenSearch cluster deployments with low replica counts, such as those used for log analytics.
 
+## Remote store
+
+As of OpenSearch 2.10, segment replication now supports two methods for storing replica shards:
```

Comment:
I am not sure if "two methods for storing replica shards" is the right term. May be 2 methods of hydrating data from primary shards to replica shards.

Reply:
Lets be more generic and say "...two methods for segment replication."